src/nls/en/strings.json

```diff
@@ -94,7 +94,11 @@
                 "HEADLINE": "Design Space",
                 "BODY_FIRST": "Twitter",
                 "BODY_SECOND": "GitHub",
-                "BODY_THIRD": "Forum"
+                "BODY_THIRD": "Forum",
+                "BODY_ADVERTISEMENT_1": "You seem like the type who likes to check out betas. We've been hard at work building a new end-to-end solution for UX designers, including great integration with Photoshop. We're happy to share that",
+                "BODY_ADVERTISEMENT_2": "Adobe Experience Design CC",
+                "BODY_ADVERTISEMENT_3": "is now in public beta.",
```

Comment:
Sorry for not being clear. React can render an array of mixed objects. I think we can make use of that by passing in an arrary that contains pure string and wrapped string:

```jsx
var description = nls.localize("strings.FIRST_LAUNCH.SLIDES.7.BODY_ADVERTISEMENT").split("{ADOBE_XD_CC}"),
    name = (<em>nls.localize("strings.FIRST_LAUNCH.SLIDES.7.ADOBE_XD_CC")</em>);

description.splice(1, 0, name); // insert the name

return (
  <div>
    {description}
  </div>
);
```